contractcourt/chain_watcher.go

```diff
@@ -393,6 +393,96 @@ func isOurCommitment(localChanCfg, remoteChanCfg channeldb.ChannelConfig,
 	return false, nil
 }
 
+// chainSet includes all the information we need to dispatch a channel close
+// event to any subscribers.
+type chainSet struct {
+	// remoteStateNum is the commitment number of the lowest valid
+	// commitment the remote party holds from our PoV.
+	remoteStateNum uint64
```

Comment:
nit: maybe mention that we determine if the remote is playing a different state (i.e. one ahead, breach etc) by comparing to this value?